<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2E3440;
color: #D8DEE9;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #81A1C1;
    font-size: 16pt;
    font-weight: bold;
    qproperty-alignment: AlignCenter;
}</string>
     </property>
     <property name="text">
      <string>Registration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditUsername">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #3B4252;
    color: #E5E9F0;
    border: 1px solid #4C566A;
    border-radius: 4px;
    padding: 8px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #3B4252;
    color: #E5E9F0;
    border: 1px solid #4C566A;
    border-radius: 4px;
    padding: 8px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditStatus">
     <property name="styleSheet">
      <string notr="true">QTextEdit {
    background-color: #3B4252;
    color: #A3BE8C;
    border: 1px solid #4C566A;
    border-radius: 4px;
    padding: 8px;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonRegister">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #5E81AC;
    color: white;
    border-radius: 4px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #81A1C1;
}</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>